rotkehlchen/db/dbhandler.py

```diff
@@ -1396,6 +1413,9 @@ def purge_module_data(self, module_name: Optional[ModuleName]) -> None:
             self.delete_yearn_vaults_data()
         elif module_name == 'loopring':
             self.delete_loopring_data()
+        elif module_name == 'eth2':
+            self.delete_eth2_deposits()
```

Comment:
It seems to work fine, when tested, but why split it in two different functions? For now we call them both together so just keep it as one. `delete_eth2_data()`